lib/grppc.gi

```diff
@@ -1852,6 +1852,96 @@ InstallGlobalFunction( GapInputPcGroup, function(U,name)
 
 end );
 
+#############################################################################
+##
+#F  PrintPcPresentation( <grp>, <commBool> )
+##
+##  Display the pc relations of a pc group.
+##  If <commBool> is true, then the commutator presentation is printed,
+##  otherwise the power presentation.
+##  Trivial commutators / powers are not printed.
+##  The generators are named "g<i>".
+##  The returned boolean indicates if there are commutating generators.
+##
+InstallGlobalFunction( PrintPcPresentation, function(G, commBool)
+    local pcgs, n, F, gens, i, pis, exp, t, h, rel, commPower, j, trivialCommutators;
+
+    pcgs:=Pcgs(G);
+    n:=Length(pcgs);
```

Comment:
This is not a request for change, just wanted to point something out: the above two lines clearly come from another source than the following three lines, based on their formatting... Overall, this function mixes several code formatting styles (e.g. `Foo( arg )` vs. `Foo(arg)`)

Reply:
@fingolfin What is the GAP coding style? As I didn't knew I copied code without changing its formating and coded the rest like it seems to make sense to me.

Reply:
There is no official coding style (and trying to set one is one of these things were I expect a lot of pain and blood for little gain). Many people seem to take the formatting which GAP itself uses when printing a function as reference, but usually with some adjustments (e.g. aligning some things across multiple lines, as below).

Anyway, I am not asking for a specific style to be followed -- just that the code within a given function be self-consistent. That's note the case here.

Reply:
I'm going to fix this tomorrow by adding a new commit, so that you can do a reintegration merge afterwards.

Reply:
I already merged this PR, didn't want to hold it hostage over some whitespace differences :-) I only pointed this formatting bit out again to make you aware of it; as it is, lots of GAP code is less well formatted than what is in this PR...